cost/google/idle_vm_recommendations/google_vm_recommendations.pt

```diff
@@ -400,65 +401,74 @@ script "js_instance_cost_mapping", type:"javascript" do
+            var cur = "$"
           }
-          strNano = "."+recommendation.costNanos
-          combine = recommendation.costUnits + parseFloat(strNano)
-          monthsDuration = parseFloat(recommendation.duration.replace("s",""))/2628288
-          monthlySavings = combine / monthsDuration
-          total = total+monthlySavings
-          vm['savings']= (Math.round( monthlySavings * 1000) / 1000)
           vm['savingsCurrency'] = cur
-          vm['priority'] = recommendation['priority']
-          vm['region'] = recommendations['region']
-          vm['primaryImpact'] = recommendation['primaryImpact']
-          vm['recommenderSubtype'] = recommendation['recommenderSubtype']
-          vm['state'] = recommendation['state']
-          vm['service'] = "Compute"
-          tags = []
-          if (vm['tags'] != null) {
-            if( typeof vm['tags'] == 'object') {
-              Object.keys(vm.tags).forEach(function(key) {
-                if (vm.tags[key] == null || vm.tags[key]=="") {
-                  vm.tags[key] = "null"
-                }
-                tags.push(String(key)+'='+String(vm.tags[key]))
-              });
-            } else {
-              tags.push(vm["tags"].replace(":","=null"))
-            }
+        } else {
+          var cur = "$"
+          var separator = ","
+        }
+        if (recommendation.costNanos<0) {
+          recommendation.costNanos = recommendation.costNanos * -1
+        }
+        if (recommendation.costUnits<0) {
+          recommendation.costUnits = recommendation.costUnits * -1
+        }
+        var cost_nano = recommendation.costNanos * Math.pow(10, -9)
+        var combine = Number(recommendation.costUnits) + Number(parseFloat(cost_nano))
+        monthsDuration = parseFloat(recommendation.duration.replace("s",""))/2628288
+        monthlySavings = combine / monthsDuration
+        total = total+monthlySavings
+        vm['savings']= (Math.round( monthlySavings * 1000) / 1000)
+        vm['savingsCurrency'] = cur
+        vm['priority'] = recommendation['priority']
+        vm['region'] = recommendations['region']
+        vm['primaryImpact'] = recommendation['primaryImpact']
+        vm['recommenderSubtype'] = recommendation['recommenderSubtype']
+        vm['state'] = recommendation['state']
+        vm['service'] = "Compute"
+        tags = []
+        if (vm['tags'] != null) {
+          if( typeof vm['tags'] == 'object') {
+            Object.keys(vm.tags).forEach(function(key) {
+              if (vm.tags[key] == null || vm.tags[key]=="") {
+                vm.tags[key] = "null"
+              }
+              tags.push(String(key)+'='+String(vm.tags[key]))
+            });
+          } else {
+            tags.push(vm["tags"].replace(":","=null"))
           }
-          vm['tags'] = tags
-          instances.push(vm)
         }
-      })
+        vm['tags'] = tags
+        instances.push(vm)
+      }
     })
     if (instances.length != 0){
       if(count){
```

Comment:
I can't select the line but, on Line 540 could we add a field for "primaryImpact category"? Like so:

```ruby 
field "primaryImpactCategory" do
  label "Recommendation Primary Impact Category"
  path "primaryImpact.category"
 end
```

Again, this may apply to the other 3 policies as well